pq-crypto/bike/decode.c

```diff
@@ -47,37 +47,37 @@ typedef ALIGN(16) struct decode_ctx_s
 {
     // Count the number of unsatisfied parity-checks:
     ALIGN(16) uint8_t upc[N_DDQWORDS_BITS];
-    
+
     e_t black_e;
     e_t gray_e;
     int delta;
     uint32_t threshold;
 } decode_ctx_t;
 
-void split_e(OUT split_e_t *split_e_, IN const e_t *e)
+void split_e(OUT split_e_t *split_e, IN const e_t *e)

```

Comment:
This appears to revert an earlier commit to eliminate shadowing warnings. How was this warning resolved?